FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/SsnUserInfo.java

```diff
@@ -12,20 +12,16 @@
  */
 public class SsnUserInfo {
 
-    private final String country;
+    private final Country country;
     private final String ssn;
 
-    public static SsnUserInfo create(Country country, String ssn) {
-        return new SsnUserInfo(country.getCountryCode(), ssn);
-    }
-
     @JsonCreator
-    private SsnUserInfo(@JsonProperty("country") String country, @JsonProperty("ssn") String ssn) {
+    public SsnUserInfo(@JsonProperty("country") Country country, @JsonProperty("ssn") String ssn) {
```

Comment:
Why we set this public?

Reply:
Reverted. And added public static create(Country, String).